src/setupTests.ts

```diff
@@ -1,5 +1,4 @@
-import * as Enzyme from 'enzyme';
-require('jest-localstorage-mock');
-const Adapter = require('enzyme-adapter-react-16');
-
-Enzyme.configure({ adapter: new Adapter() });
+import { configure } from 'enzyme';
+import * as ReactSixteenAdapter from 'enzyme-adapter-react-16';
+const adapter = ReactSixteenAdapter as any;
```

Comment:
Is this really required?

Reply:
I believe it is. I copied this code from the Enzyme setup.

Reply:
@josejulio removed the `as any` and everything still works fine.

Reply:
Travis is complaining about something :(